src/webgpu/api/validation/capability_checks/limits/maxBindingsPerBindGroup.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { kLimitBaseParams, makeLimitTestGroup } from './limit_utils.js';
+
+const limit = 'maxBindingsPerBindGroup';
+export const { g, description } = makeLimitTestGroup(limit);
+
+g.test('createBindGroupLayout,at_over')
+  .desc(`Test using at and over ${limit} limit`)
+  .params(kLimitBaseParams)
+  .fn(async t => {
+    const { limitTest, testValueName } = t.params;
+    await t.testDeviceWithRequestedLimits(
+      limitTest,
+      testValueName,
+      async ({ device, testValue, shouldError }) => {
+        await t.expectValidationError(() => {
+          device.createBindGroupLayout({
+            entries: [
+              {
+                binding: testValue - 1,
+                visibility: GPUShaderStage.VERTEX,
+                buffer: {},
+              },
+            ],
+          });
+        }, shouldError);
+      }
+    );
+  });
```

Comment:
Similar situation with 'auto' layout, again not sure if it should be caught in creating the pipeline or the shader module

Reply:
added

Reply:
added